Remove an old link to Google's documentation (#2513)
segmentio/segment-docs: src/connections/destinations/catalog/firebase/index.md

### **Conversion Tracking and Adwords Conversions**

> Firebase is now Google's recommended method for reporting conversions to Adwords! To do so, simply track the conversion events as you normally would with Segment and Segment will send them through to Firebase. 

Reviewer: ```suggestion
Firebase is Google's recommended method for reporting conversions to Adwords. To use Firebase, track the conversion events as you normally would with Segment and Segment will send them through to Firebase. 
```
